taxi/views.py

```diff
@@ -44,6 +45,46 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
```

Comment:
You don't need to use template_name here, because your name is the same as default.(checklist, line 9)